### PR TITLE
bump go.d.plugin to v0.37.1

### PR DESCRIPTION
**Pull request by ilyam8**
##### Summary

Fixes: #13549
Closes: #13550

bump go.d.plugin version to [v0.37.1](https://github.com/netdata/go.d.plugin/releases/tag/v0.37.1)

v0.37.1 contains a patch that changes the following charts:
- [galera_cluster_status](https://github.com/netdata/go.d.plugin/blob/faa01a27a40818eeef5b849ae599353ab000dc48/modules/mysql/charts.go#L751-L759) => [new version](https://github.com/netdata/go.d.plugin/blob/84fb4f433b1d9b2556361e2bdab2efac4367b424/modules/mysql/charts.go#L751-L762).
- [galera_cluster_state](https://github.com/netdata/go.d.plugin/blob/faa01a27a40818eeef5b849ae599353ab000dc48/modules/mysql/charts.go#L762-L771) => [new version](https://github.com/netdata/go.d.plugin/blob/84fb4f433b1d9b2556361e2bdab2efac4367b424/modules/mysql/charts.go#L764-L778).

TL;DR The change is switching from encoding the state/status with an integer value to a dimension per state/status.

In addition, this PR adjusts corresponding MySQL alarms and dashboard info.

##### Component Name

`packaging/`

##### Test Plan

- install this branch, ensure there are no errors, and go.d.plugin version is 0.37.1.
- check Galera charts and alarms.

##### Additional Information

### CODE REVIEW

**Review by thiagoftsm — APPROVED** [review]
My modules are working as expected, LGTM!

File: health/health.d/mysql.conf [hunk 1]
```diff
@@ -136,20 +136,29 @@ component: MySQL
...
+       on: mysql.galera_cluster_state
+    class: Errors
+     type: Database
+component: MySQL
+     calc: $undefined + $joining + $error
+    every: 10s
+     crit: $this != nan AND $this != 0
+    delay: up 30s down 5m multiplier 1.5 max 1h
+     info: galera node state is either Undefined or Joining or Error.
+       to: dba
```
Comment (MrZammler) [inline health/health.d/mysql.conf:161]: Hey Ilya!

We should have a way to skip maybe both `calc` and `lookup`, and use directly the dimensions directly in `crit` and `warn`...

**Review by MrZammler — APPROVED** [review]

File: health/health.d/mysql.conf (hunk 1 above)
Comment (ilyam8) [reply health/health.d/mysql.conf:161]: Is that a question or a statement?

Comment (MrZammler) [reply health/health.d/mysql.conf:161]: A thought :-)

Comment (ilyam8) [reply health/health.d/mysql.conf:161]: I hope you'd say a commitment 😄
